config/routes.rb

```diff
@@ -6,5 +6,9 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
+  root 'home#index'
+
+  get '/home' => 'home#index'
```

Comment:
what is the reason to use the same content in `root` and `/home`

Reply:
done  :+1: